<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="msgBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="settingLayout">
     <item>
      <widget class="QLabel" name="serverPortlabel">
       <property name="text">
        <string>服务器端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serverPortEdit">
       <property name="text">
        <string>8000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="serverPortButton">
       <property name="text">
        <string>设定</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="msgEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chatRecordButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>聊天记录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
